plugins/woocommerce/src/Internal/Admin/Orders/ListTable.php

```diff
@@ -273,6 +274,17 @@ private function set_status_args() {
 		$this->order_query_args['status'] = $query_statuses;
 	}
 
+	/**
+	 * Implements order search.
+	 */
+	private function set_search_args(): void {
+		$search_term = trim( sanitize_text_field( wp_unslash( $_REQUEST['s'] ?? '' ) ) );
+
+		if ( ! empty( $search_term ) ) {
+			$this->order_query_args['s'] = $search_term;
```

Comment:
I believe we should set `has_filter = true` here. Otherwise, when no results are found the list table will display the "blank state" placeholder instead of "No items found". To repro, just search for anything that you know won't produce matches.

Reply:
You're right—fixing.

Reply:
Thanks for catching that 😅 ... addressed in https://github.com/woocommerce/woocommerce/pull/34421/commits/b44273bf78c63ed0d19d4d7cbb60f0967f2b449c.